src/components/attendee/AttendeeLayout.module.scss

```diff
@@ -69,3 +70,7 @@ $-z-index--popover: layer.below($-z-index--ui);
     pointer-events: all;
   }
 }
+
+.blur {
+  filter: blur(100px) contrast(60%);
```

Comment:
[NOTE]
According to @colinhowe this blur on filter and/or the backdrop-filter is making animation slow. We should look into replacing with translucent PNG backgrounds (small images can be tile-able)